src/Command/ProjectCommands.php

```diff
@@ -86,6 +91,9 @@ private function getPhpUnitExecutable()
         return $phpunit_executable;
     }
 
+    /**
+     *
```

Comment:
```suggestion
     * Ensure current git branch is not in a detached state.
     *
     * @return bool
     *   Flag for whether the current branch branch is detached.
```
Just adding documentation.